infra/modules/azure_core_infra/README.md

```diff
@@ -0,0 +1,84 @@
+# Core Infrastructure module
+The module provisions all the resources required for the initial configuration of a subscription, which can be utilized for testing or other purposes.
```

Comment:
what does it means "other purposes"? which purposes? is this supposed to be used for production environments as well?